src/components/icon.js

```diff
@@ -7,13 +7,15 @@ const Icon = props => {
   const { d, size = '1em', label, style: styles } = props
 
   return (
-    <span className={style.root} style={styles} role="figure">
+    <span className={style.root} style={styles} role="img" aria-label="Hello Friend">
```

Comment:
Label should be as a prop. Because `Icon` component is a template for any `svg` flatten icon by passing `d` prop. (line 18-20)